R/f7VirtualList.R

```diff
@@ -71,11 +71,30 @@ f7VirtualList <- function(id, items, rowsBefore = NULL, rowsAfter = NULL,
 #'
 #' \code{f7VirtualListItem} is an item component for \link{f7VirtualList}.
 #'
-#' @inheritParams f7ListItem
+#' @param ... Item text.
+#' @param id Optional id for item.
+#' @param title Item title.
+#' @param subtitle Item subtitle.
+#' @param header Item header.
+#' @param footer Item footer.
+#' @param href Item external link.
+#' @param media Expect \link{f7Icon} or \code{img}.
+#' @param right Right content if any.
+#' @param routable Works when href is not NULL. Default to FALSE. If TRUE,
+#' the list item may point to another page, but we recommend using \link{f7List}
+#' and \link{f7ListItem} instead. See \link{f7MultiLayout}. Can also be used in
+#' combination with href = "#" to make items appear as links, but not actually
+#' navigate anywhere, which is useful for custom click events.
+#'
```

Comment:
@DivadNojnarg I added a `routable` argument here, which has in strict sense no relevance for a virtual list. But it is handy to use in combination with `href = "#"` because based on this routable arg, we can remove the external class. I thought it was best to align with `f7ListItem()`.

Now, in combination with the id in both list items, it is relatively user-friendly to add custom click events if desired. 

What are your thoughts on this? I don't want make things too messy or too unclear, but I guess it doesn't harm much but will be very helpful in specific cases? Could support this in the lists vignette..